features/importers/venepaikka_harbors/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "tag_config": {"id": "ahti:tag:island", "name": "saaristo"},
+  "category_config": {"id": "ahti:category:island", "name": "Saaret"},
```

Comment:
shouldn't these be:
```
"tag_config": {"id": "ahti:tag:harbor", "name": "satama"},
"category_config": {"id": "ahti:category:harbor", "name": "Satamat"}
```
?

Reply:
They should. :sob: Looks like some version of copy&paste error.